en/user/articles/120-search/06-search-results/03-custom-filters.markdown

```diff
@@ -48,6 +48,10 @@ click _Configuration_.
 field to be filtered (for example, `title`). You won't need this if the Filter
 Query Type is set to a type that doesn't require a field, such as _Regexp_.
 
+> The Query String and Script queries do not require a Filter Field to be set.
+> All other queries require at least one field. 
+<!--Note: Multi Match and Simple Query String take an array of fields accoring to the Elasticsearch docs, but our config doesn't seem to support it afaict.? -->
+
```

Comment:
@lipusz, I thought you might like to give this a check. It's what I was asking about in #help-search